src/Features/Workflows/WorkflowCard/UpdateWorkflow/UpdateWorkflow.tsx

```diff
@@ -38,15 +38,15 @@ const UpdateWorkflow: React.FC<UpdateWorkflowProps> = ({ teamId, workflowId, onC
         containerClassName: styles.container,
       }}
       modalHeaderProps={{
-        title: "Update .json file",
+        title: "Update Workflow",
```

Comment:
Unrelated updates to this copy that I came across when testing and wanted to update to make more sense after receiving a few complaints.

Reply:
I agree that it could be confusing if you are not familiar with the json structure of a workflow and the import/export process. I feel like having `edit workflow` and `update workflow` menu options could also be a little confusing. Perhaps `Overwrite workflow`? Or something to help make it distinct that this is how you can import json and create a new revision?